nautobot/extras/forms/base.py

```diff
@@ -62,3 +64,9 @@ class NautobotBulkEditForm(
     RelationshipModelBulkEditFormMixin,
 ):
     """Base class for bulk-edit forms for models that support relationships, custom fields and notes."""
```

Comment:
Are there any models that have a BulkEditForm that isn't a a NautobotBulkEditForm subclass?